moveit_ros/planning/robot_model_loader/CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ ament_target_dependencies(${MOVEIT_LIB_NAME}
   urdf
   Boost
   moveit_core
+  moveit_msgs
```

Comment:
This CMakeLists is missing the `find_package` entries!

Reply:
This is one of the many MoveIt packages that uses subdirectories, so the `find_package` is in the CMakeLists one level up.